<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StellaServer</class>
 <widget class="QStackedWidget" name="StellaServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StackedWidget</string>
  </property>
  <property name="currentIndex">
   <number>2</number>
  </property>
  <widget class="QWidget" name="page">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>121</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::MinimumExpanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>204</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="lblConnect">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string notr="true">TEXT</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::MinimumExpanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>204</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="btnAbortConnection">
      <property name="text">
       <string>Abort</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <spacer name="verticalSpacer_6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>121</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="pwmchannels">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>EEProm:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnLoadEEPROM">
           <property name="toolTip">
            <string>Load values from stella eeprom</string>
           </property>
           <property name="text">
            <string>Load</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/document-open.png</normaloff>:/icons/document-open.png</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnSaveEEPROM">
           <property name="toolTip">
            <string>Save current values to eeprom</string>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/document-save.png</normaloff>:/icons/document-save.png</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_4">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QToolButton" name="btnTestcase">
           <property name="toolTip">
            <string>Testcases like Fading up/down</string>
           </property>
           <property name="text">
            <string>Testcases</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/flag.png</normaloff>:/icons/flag.png</iconset>
           </property>
           <property name="popupMode">
            <enum>QToolButton::InstantPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QToolButton" name="btnAbout">
           <property name="toolTip">
            <string>About stella server</string>
           </property>
           <property name="text">
            <string>About stella server</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/help-hint.png</normaloff>:/icons/help-hint.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnClose">
           <property name="text">
            <string>Close stella server connection</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/dialog-close.png</normaloff>:/icons/dialog-close.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>395</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Predefined sets:</string>
         </property>
         <property name="buddy">
          <cstring>predefinedlist</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="predefinedlist">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRemovePredefined">
         <property name="text">
          <string>Remove</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/icons/edit-delete.png</normaloff>:/icons/edit-delete.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSavePredefined">
         <property name="text">
          <string>Save</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/icons/document-save.png</normaloff>:/icons/document-save.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="page_2">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="toolTip">
         <string>Beats per minute</string>
        </property>
        <property name="text">
         <string>BPM:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBeat">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="frame_7">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QToolButton" name="btnAbout_2">
           <property name="toolTip">
            <string>About stella server</string>
           </property>
           <property name="text">
            <string>About stella server</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/help-hint.png</normaloff>:/icons/help-hint.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnClose_2">
           <property name="text">
            <string>Close stella server connection</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/icons/dialog-close.png</normaloff>:/icons/dialog-close.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>260</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <action name="actionBroadcastChanges">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Stella Broadcast Changes</string>
   </property>
   <property name="toolTip">
    <string>Broadcast changes 1s after the last change</string>
   </property>
  </action>
  <action name="actionFade_Test_1">
   <property name="text">
    <string>Fade Test 1 (Up)</string>
   </property>
  </action>
  <action name="actionFade_Test_2">
   <property name="text">
    <string>Fade Test 2 (Down)</string>
   </property>
  </action>
  <action name="actionSet_speed_rate">
   <property name="text">
    <string>Set speed rate</string>
   </property>
  </action>
  <action name="actionFade_Test_3">
   <property name="text">
    <string>Fade test 3 (Flashy)</string>
   </property>
  </action>
  <action name="actionFade_Test_4">
   <property name="text">
    <string>Regression Test 4 (1-0 Flicker)</string>
   </property>
  </action>
  <action name="actionChannelCount">
   <property name="text">
    <string>Visible Channels</string>
   </property>
  </action>
  <action name="actionRefetch">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/arrow-left.png</normaloff>:/icons/arrow-left.png</iconset>
   </property>
   <property name="text">
    <string>Refetch</string>
   </property>
  </action>
  <action name="actionResend">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/arrow-right.png</normaloff>:/icons/arrow-right.png</iconset>
   </property>
   <property name="text">
    <string>Resend</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
